tests/Functional/Runner/ChunkFileTest.php

```diff
@@ -12,6 +12,20 @@
 
 class ChunkFileTest extends BaseIntegrationTestCase
 {
+    public function testChunkedPlusTestSuiteOptions(): void
+    {
+        $this->setOption('chunk-size', '2');
+        $this->setOption('testsuite', 'stubs');
+        $this->setOption('debug', '1');
+        $this->loadContainer();
+
+        $this->executeRunner();
```

Comment:
The issue is in `\Paraunit\Command\ParallelCommand::addPHPUnitOptions` which it should not be covered by this test case.